scripts/test-extensions-unit.js

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const argv = require('minimist')(process.argv.slice(2));
+const execSync = require('child_process').execSync;
+const tmp = require('tmp');
+const fs = require('fs-extra');
+const path = require('path');
+const os = require('os');
+
+let extensions = [
+	'admin-tool-ext-win',
+	'agent',
+	'azurecore',
+	'cms',
+	'dacpac',
+	'schema-compare',
+	//'mssql',
+	'notebook',
+	'resource-deployment',
+	'machine-learning-services',
+	'sql-database-projects'];
+
+// validate args
+
+if (argv._.length > 0) {
```

Comment:
You don't have to manually parse these https://github.com/microsoft/azuredatastudio/blob/master/scripts/code-web.js#L22

Reply:
You could use optimist too (its already a dep) https://www.npmjs.com/package/optimist not sure the full difference but it looks like it might have more features.

Reply:
Dunno about optimist 
![image](https://user-images.githubusercontent.com/1609827/80780064-749fcc80-8b22-11ea-84fe-5650109b144c.png)

[Yargs](https://github.com/yargs/yargs) looks interesting, so far as automatically generating useful helptext, though.  Looks like it has lots of the features I like in powershell's arg framework.

Reply:
I would prefer using a framework we already have in the project rather than adding another, you can still do all the parsing with minimist so.